web_widget_image_webcam/static/src/js/webcam_widget.js

```diff
@@ -0,0 +1,106 @@
+/*
+    Copyright 2016 Siddharth Bhalgami <siddharth.bhalgami@techreceptives.com>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+*/
+odoo.define('web_widget_image_webcam.webcam_widget', function(require) {
+    "use strict";
+
+    var core = require('web.core');
+    var Model = require('web.DataModel');
+
+    core.form_widget_registry.get("image").include({
+
+        render_value: function () {
+            this._super();
+
+            var self = this,
+                img_data;
+
+            // ::webcamjs:: < https://github.com/jhuckaby/webcamjs >
+            // Webcam: Set Custom Parameters
+            Webcam.set({
+                width: 320,
+                height: 240,
+                dest_width: 320,
+                dest_height: 240,
+                image_format: 'jpeg',
+                jpeg_quality: 90,
+                force_flash: false,
+                fps: 45,
+            });
+
+            self.$el.find('.oe_form_binary_file_clear').removeClass('col-md-offset-5');
+
+            new Model('ir.config_parameter').query(['key', 'value']).
+            filter([['key', '=', 'web_widget_image_webcam.flash_fallback_mode']]).
+            all().then(function(default_flash_fallback_mode) {
```

Comment:
Better just call to [`get_param`](https://github.com/odoo/odoo/blob/7682760dcad232829fbb5d4221f66a1ebb9d6c91/openerp/addons/base/ir/ir_config_parameter.py#L53).